kotlin/android/app/src/main/java/dev/firezone/android/features/session/backend/SessionManager.kt

```diff
@@ -18,13 +19,14 @@ internal class SessionManager @Inject constructor(
         try {
             val config = getConfigUseCase.sync()
 
-            if (config.portalUrl != null && config.jwt != null) {
-                Log.d("Connlib", "portalUrl: ${config.portalUrl}")
-                Log.d("Connlib", "jwt: ${config.jwt}")
+            Log.d("Connlib", "accountId: ${config.accountId}")
+            Log.d("Connlib", "token: ${config.token}")
 
+            if (config.accountId != null && config.token != null) {
                 sessionPtr = Session.connect(
+                    -1, // TODO: get fd from VpnService. See VpnBuilder#establish().detachFd()
```

Comment:
@conectado Here is where it would be convenient to pass in the file descriptor in a single line